.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+default_language_version:
+  python: python3.10
+
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
```

Comment:
We've got `21.10b0` in the dev requirements, could you bump both to `22.12.0`? Probably best to start from the current version!